### PR TITLE
Add Lewis' solutions

### PR DESCRIPTION
**Pull request by BenH14**

### CODE REVIEW

File: minitask1/Lewis/TurtlebotDriving.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        # Stop message to stop the Turtlebot
+        stopMessage = Twist()
+        stopMessage.linear.x = 0
+        stopMessage.angular.z = 0
+
+        # Variables to keep track of square drawing progress
+        sideCount = 0
+        turnCount = 0
+
+        while not rospy.is_shutdown() and sideCount != 4 and turnCount != 4:
```
Comment (bhenaghan) [inline minitask1/Lewis/TurtlebotDriving.py:34]: I think a condition like `sideCount < 4` is a lot clearer than `sideCount != 4` as we know that sidecount should be increasing. In this case it doesn't make any functional difference but I think its clearer and simpler

**Comment by lew101** [comment]
I've just realised, I've forgotten to add the tracing.

The open-loop has a habit of undershooting for the first straight but the rest of the square works nicely.
The closed-loop works well, just missing the tracing. 

I was trying to figure out a better way of doing the 4 sides but it seems okay

File: minitask1/Lewis/TurtlebotDriving_closedLoop.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+# Drives the Turtlebot in a 1m sqaure using open-loop control.
+class TurtlebotDriving_closedLoop():
+    
+    def __init__(self):
+        self.ongoingTheta = 0
+        self.currentTheta = 0
+        self.x = 0
+        self.y = 0
+        self.pub = rospy.Publisher('cmd_vel', Twist, queue_size = 10)
```
Comment (bhenaghan) [inline minitask1/Lewis/TurtlebotDriving_closedLoop.py:18]: Shouldn't have spaces around named params

File: minitask1/Lewis/TurtlebotDriving_closedLoop.py [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+from geometry_msgs.msg import Twist
+from nav_msgs.msg import Odometry
+from math import radians
+from tf.transformations import euler_from_quaternion
+
+# Drives the Turtlebot in a 1m sqaure using open-loop control.
+class TurtlebotDriving_closedLoop():
+    
+    def __init__(self):
```
Comment (bhenaghan) [inline minitask1/Lewis/TurtlebotDriving_closedLoop.py:13]: I don't think we should have all this code in the `__init__`

File: minitask1/Lewis/TurtlebotDriving_closedLoop.py [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+        leftMessage = Twist()
+        leftMessage.angular.z = radians(10)
+
+        # Stop message to stop the Turtlebot
+        stopMessage = Twist()
+        stopMessage.linear.x = 0
+        stopMessage.angular.z = 0
+
+        # Variables to keep track of square drawing progress
+        sideCount = 0
```
Comment (bhenaghan) [inline minitask1/Lewis/TurtlebotDriving_closedLoop.py:36]: `sideCount` and `turnCount` aren't ever used

**Pull request merged**